system/help/help.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Creates an info popup.
+ *
+ * @param clickEvent Element, click event, used to determine position of popup
+ * @param key string, the key for with to get the info
+ * @param language string
+ * @param infoButton Element, the info button for which to make an info popup
+ */
+function toggleInfoPopup(clickEvent, key, language, infoButton) {
+	// If it has a clicked state, close the popup and remove the state.
+	var $infoButton = $(infoButton);
+	if ($infoButton.hasClass('clicked')) {
+		$infoButton.trigger('closeMe');
+		return;
+	}
+
+	// Directly set the clicked class, this also prevents the spamming of the system before the result is returned.
+	$infoButton.addClass('clicked');
+
+	// Create an empty popup.
+	var $helpPopup = $('<div></div>').appendTo('body');
+x§
```

Comment:
Is this one intentional? You see to have added in the last force-push?

Reply:
Oops. :-)